src/Core/src/Handlers/View/ViewHandler.iOS.cs

```diff
@@ -0,0 +1,76 @@
+﻿using CoreAnimation;
+using CoreGraphics;
+using Microsoft.Maui.Graphics;
+using NativeView = UIKit.UIView;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ViewHandler
+	{
+		CALayer? _layer;
+		bool _isInteractive;
+		CGPoint? _originalAnchor;
+		Rectangle? _lastBounds;
+
+		partial void SettingDefault(NativeView? nativeView)
+		{
+			_layer = nativeView?.Layer;
+			_isInteractive = nativeView?.UserInteractionEnabled ?? false;
+			_originalAnchor = _layer?.AnchorPoint;
+			_lastBounds = Rectangle.Zero;
+		}
```

Comment:
This will fire only once per type, so this will end up being fired only once per app - probably - and thus these values will be the very fires view created and then reused for all views. Maybe this should be in ConnectHandler and then nulled out in DisconnectingHandler?

Reply:
Moved to `ConnectHandler` and `DisconnectHandler`